config/triplea/schemas/lobby_server.json

```diff
@@ -10,7 +10,7 @@
     "properties": {
       "version": {
         "description": "The lobby server engine version",
-        "type": "string",
+        "type": [ "string", "number" ],
```

Comment:
Why not just change the regex to something like `^\\d+(\\.\\d+){0,3}$`?  A `number` will allow  stuff like `-1.10` through the validator.